setup.py

```diff
@@ -210,12 +211,21 @@ def run(self):
                 build_args.extend(['--', '-j', str(self.jobs)])
             subprocess.check_call(build_args)
 
+            install_args = [CMAKE, '--install', os.curdir]
+            if WINDOWS:
+                install_args.extend(['--config', build_type])
+            subprocess.check_call(install_args)
+
 
 class build_py(setuptools.command.build_py.build_py):
     def run(self):
         self.run_command('create_version')
         self.run_command('cmake_build')
 
+        self.copy_tree(os.path.join(CMAKE_BUILD_DIR,"install","include"),
+                os.path.join(TOP_DIR, self.build_lib, "onnx", "include"))
+        self.copy_tree(os.path.join(CMAKE_BUILD_DIR,"install","lib"),
```

Comment:
```suggestion
        self.copy_tree(os.path.join(CMAKE_BUILD_DIR, "install", "lib"),
```
Add a space after , to meet flake8. L225 needs the same fix.